modules/ROOT/pages/queries/expressions.adoc

```diff
@@ -0,0 +1,79 @@
+= Cypher expressions
+:description: This page explains which expressions are allowed in Cypher.
+
+This page contains examples of allowed expressions in Cypher.
+
+[NOTE]
+====
+Most expressions in Cypher evaluate to `null` if any of their inner expressions are `null`.
+Notable exceptions are the operators `IS NULL`, `IS NOT NULL`, and the xref:values-and-types/type-predicate.adoc[type predicate expressions].
+====
```

Comment:
I think this is quite a subtle remark. I'd move it at least to the bottom of `General`.